src/App.js

```diff
@@ -47,7 +50,7 @@ class App extends Component {
     return (
       <main>
         < Nav returnHome={this.returnHome}/>
-        {(this.state.errors) ? < Error /> : (this.state.movieSummary) ? < MovieSummary movieToView={this.state.movieToView}/> : < PosterContainer movies={this.state.movies} showMovieSummary={this.showMovieSummary}/>}
+        {(this.state.errors) ? < Error /> : (this.state.movieSummary) ? < MovieSummary movieToView={this.state.movieToView}/> : < HomePage moviesProp={this.state.movies} showMovieSummary={this.showMovieSummary}/>}
```

Comment:
This may be to complex of a ternary perhaps this could be refactored to be easier to read.